charts/karpenter/templates/webhook/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: karpenter-webhook
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+    - port: 443
+      targetPort: webhook
+  selector:
+    karpenter: webhook
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karpenter-webhook
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      karpenter: webhook
+  template:
+    metadata:
+      labels:
+        karpenter: webhook
+    spec:
+      priorityClassName: system-cluster-critical
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      containers:
+        - name: webhook
+          image: {{ .Values.webhook.image }}
+          resources:
+            limits:
+              cpu: 100m
+              memory: 50Mi
```

Comment:
Wondering where does 50Mi come from?

Reply:
I played with this and got OOMs until ~25Mi. I doubled it, since this webhook shouldn't be under much load at all (i.e. only when provisioners are created/deleted)

Reply:
Great, thanks